frontend/public/components/storage/create-pvc.tsx

```diff
@@ -250,6 +258,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
           Use label selectors to define how storage is created
         </p>
       </div>
+      <label className="control-label" htmlFor="volume-mode">
+        Volume Mode
+      </label>
+      <div className="form-group">
+        {volumeModeRadios.map((radio) => (
+          <RadioInput
+            {...radio}
+            key={radio.value}
+            onChange={handleVolumeMode}
+            inline={true}
```

Comment:
nit: this can just be

```suggestion
            inline
```